parm/wflow/verify_det.yaml

```
@@ -136,49 +136,60 @@ metatask_GridStat_REFC_RETOP_all_mems:
               age: 00:00:00:30
             text: !cycstr '{{ workflow.EXPTDIR }}/@Y@m@d@H/post_files_exist_mem#mem#.txt'
 
-metatask_PointStat_SFC_UPA_all_mems:
+metatask_PointStat:
   var:
-    mem: '{% if global.DO_ENSEMBLE %}{% for m in range(1, global.NUM_ENS_MEMBERS+1) %}{{ "%03d "%m }}{%- endfor -%} {% else %}{{ "000"|string }}{% endif %}'
-  metatask_PointStat_SFC_UPA_mem#mem#:
+    FIELD_GROUP: '{% for var in verification.VX_FIELD_GROUPS %}{% if var in ["SFC", "UPA", "AOD", "PM25", "PM10"] %}{{ "%s " % var }}{% endif %}{% endfor %}'
+    METAOBTYPE: '{% for var in verification.VX_FIELD_GROUPS %}{% if var =="SFC" or var =="UPA" %}NDAS {% elif var =="AOD" %}AERONET {% elif var =="PM25" or var =="PM10" %}AIRNOW {% endif %}{% endfor %}'
+    METAOBS_DIR: '{% for var in verification.VX_FIELD_GROUPS %}{% if var =="SFC" or var =="UPA" %}&NDAS_OBS_DIR; {% elif var =="AOD" %}&AERONET_OBS_DIR; {% elif var =="PM25" or var =="PM10" %}&AIRNOW_OBS_DIR; {% endif %}{% endfor %}'
+    METAOB_AVAIL_HRS: '{% for var in verification.VX_FIELD_GROUPS %}{% if var =="SFC" or var =="UPA" %} {{- verification.NDAS_OBS_AVAIL_INTVL_HRS }} {% elif var =="AOD" %}{{- verification.AERONET_OBS_AVAIL_INTVL_HRS }} {% elif var =="PM25" or var =="PM10" %}{{- verification.AIRNOW_OBS_AVAIL_INTVL_HRS }} {% endif %}{% endfor %}'
```

```suggestion
    METAOB_AVAIL_HRS: '{% for var in verification.VX_FIELD_GROUPS %}{% if var =="SFC" or var =="UPA" %} {{ verification.NDAS_OBS_AVAIL_INTVL_HRS }} {% elif var =="AOD" %}{{ verification.AERONET_OBS_AVAIL_INTVL_HRS }} {% elif var =="PM25" or var =="PM10" %}{{ verification.AIRNOW_OBS_AVAIL_INTVL_HRS }} {% endif %}{% endfor %}'
```

I really don't think these are necessary or have any impact on the outcome. They just add syntax. Maybe I'm wrong?